### ISSUE TITLE
Create person using a client_credential token

### ISSUE THREAD

**Issue by codeskine** [Acosix/alfresco-keycloak#28]
Hi Axel,

First i would like to thank you for this addons, I have installed modules and they works greats.

I have just a question: in our application people can be added by a non admin users and those action results in a "Permission Denied" error when request is sent to Alfresco people endpoint. 

The obviousluy solution could be to authenticate an admin user before request and use genereated token for all those requests where admin role is required . But I would like to know if it's possible also to use a  "client_credential" token for this endpoints. 

Regards,

Stefano

**Comment by AFaust** [comment]
Yes, it would be possible for an access token obtained via a client credentials grant to authenticate against Alfresco with a bearer token authentication. But a user with a matching name to the preferred_username / subject of the token still has to exist within Alfresco, or authentication fails / an ad-hoc user is created (depending whether creation of unknown people - no synched from a directory - has been allowed in Alfresco).
